test/reporter-stdout.js

```diff
@@ -0,0 +1,21 @@
+/*globals QUnit:true*/
+/*jshint node:true*/
+var QUnit = require( "../dist/qunit" );
+
+// Options: { output: "minimal" || "verbose" }
+QUnit.stdout();
+
+// Load QUnit tests
+require( "./logs" );
```

Comment:
This is the same thing already happening on Grunt [`test-on-node` task](https://github.com/jquery/qunit/blob/master/Gruntfile.js#L178).